docs/en_us/enrollment_api/source/overview.rst

```diff
@@ -0,0 +1,37 @@
+.. _edX Enrollment API Overview:
+
+################################################
+edX Enrollment API Overview
+################################################
+
+The edX Enrollment API enables you to view user and course enrollment
+information, and to enroll a user in a course.
+
+The edX Platform API uses Representational State Transfer (REST) design
+principles and supports JavaScript Object Notation (JSON) data-interchange
```

Comment:
supports *the* JavaScript (correct? or does this noun phrase not take an article?)